From d52a67cd48495404fd9cadc87921dd7147da2c75 Mon Sep 17 00:00:00 2001
From: Kyle Harding <kyle@balena.io>
Date: Sat, 13 Feb 2021 09:05:52 -0500
Subject: [PATCH] CMakeLists: patch target link libraries for buildroot

Change-type: patch
Signed-off-by: Kyle Harding <kyle@balena.io>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6202553..4fb998f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,4 +11,4 @@ link_directories(/opt/vc/lib)
 
 add_executable(fbcp main.c)
 
-target_link_libraries(fbcp bcm_host)
+target_link_libraries(fbcp bcm_host vchostif)
-- 
2.27.0

